src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -113,6 +127,12 @@ const PaymentAgentWithdraw = (() => {
                 }
             });
 
+            $txt_agents.on('focusout', () => {
+                if ($(field_ids.txt_amount).val()) {
```

Comment:
@Khalidxx can you define this element outside the listener so we don't have to keep fetching it? something like on line 129 `const $txt_amount = $(field_ids.txt_amount)` and then on line 131 `if ($txt_amount.val()) {`

Reply:
I defined it up there with the other elements